tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Surfaces.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using ColorPacket256 = VectorPacket256;
+
+using System;
+
+internal static class Surfaces
+{
+    // Only works with X-Z plane.
+    public static readonly Surface CheckerBoard =
+        new Surface(
+            delegate (VectorPacket256 pos)
```

Comment:
Why anonymous delegate instead of a lambda?

Reply:
This is to match the AoS raytracer.